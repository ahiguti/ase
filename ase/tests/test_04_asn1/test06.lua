
local Test = ASE.CPP['ASETestCPP']
-- local arr = { 10, "foobar", 99.99999 }
-- local v = 99.99999
local s = Test.ToSerial(10, "foobar", 99.99999)
print(Test.ToHEX(s))
print(#s)
local v = Test.FromSerial(s);
for i = 0, v.Length - 1 do
  print("v " .. i .. " " .. v[i])
end
local l = Test.SerialLen(s)
print(l[0])
print(l[1])
