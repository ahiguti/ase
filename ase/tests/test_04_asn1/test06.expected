9928000000000000244066666f6f62617228ee940ed6ffff5840
26
v 0 10
v 1 foobar
v 2 99.99999
true
26
